cloudify/ctx_wrappers/ctx-py.py

```diff
@@ -6,6 +6,13 @@
 import subprocess
 from collections import MutableMapping, Mapping
 
+PY2 = sys.version_info[0] == 2
+
+if PY2:
```

Comment:
separate here, because `ctx-py.py` is supposed to be a standalone script (cannot import from cloudify modules)